tools/shared.py

```diff
@@ -2298,7 +2298,10 @@ def acorn_optimizer(filename, passes, extra_info=None, return_output=False):
   # evals ctors. if binaryen_bin is provided, it is the dir of the binaryen tool for this, and we are in wasm mode
   @staticmethod
   def eval_ctors(js_file, binary_file, binaryen_bin='', debug_info=False):
-    check_call([PYTHON, path_from_root('tools', 'ctor_evaller.py'), js_file, binary_file, str(Settings.TOTAL_MEMORY), str(Settings.TOTAL_STACK), str(Settings.GLOBAL_BASE), binaryen_bin, str(int(debug_info))])
+    cmd = [PYTHON, path_from_root('tools', 'ctor_evaller.py'), js_file, binary_file, str(Settings.TOTAL_MEMORY), str(Settings.TOTAL_STACK), str(Settings.GLOBAL_BASE), binaryen_bin, str(int(debug_info))]
+    if binaryen_bin:
```

Comment:
can you add a comment explaining that `binaryen_bin` is truthy only if we are compiling wasm? That wasn't clear to me when reading this.

Reply:
never mind, comment exists!